WIP 3.0 Classes/Priest.md

```diff
@@ -509,717 +568,122 @@ You learn the *calm emotions* spell and always have it prepared. In addition, yo
 * One hope
 * One flaw
 
-The creature does not become aware that you learn this information from them.
+\columnbreak
+
+&nbsp;&nbsp;&nbsp; The creature does not become aware that you learn this information from them.
 
 Once you cast *calm emotions* in this way, you cannot do so again until you finish a short or long rest. 
 
 #### Devotion: Prayer of Mending
-*2nd-level priest feature*<br/>
+*2nd-level Holy feature*<br/>
 <div style='margin-top:-4px;'></div>
 
-As an action, you can expend a use of Devotion to place a ward of mending onto a creature you can see within 30 feet of you for 1 minute. When that creature takes damage, they regain a number of hit points equal to 2d10 plus your priest level, and gain resistance to damage for the rest of that turn, including the triggering damage. In addition, The ward then immediately jumps to a friendly creature of your choice you can see within 30 feet of the first creature. If the second creature takes damage, they regain hit points equal 1d10 plus half your priest level. After the ward has restored hit points a second time, it disappears.
+As an action, you can expend a use of Devotion to place a ward of mending onto a creature you can see within 30 feet of you for 1 minute. When that creature takes damage, they regain a number of hit points equal to 2d10 + your priest level, and gain resistance to damage for the rest of that turn, including the triggering damage. 
```

Comment:
updated wording: 
In addition, The ward then immediately moves to another friendly creature of your choice you can see within 30 feet of the first creature. If this creature takes damage, they only regain hit points equal to 1d10 plus half your priest level. After the ward has restored hit points this way, it disappears.

If the ward last for its full duration without restoring hit points, it restores hit points to the current creature by the amount it would as if it moved (2d10 plus your priest level the first time, 1d10 plus half your priest level the second time).

Specified it has to be another creature, and added only regains to imply it is the only effect it gets from being affected on the 2nd application